x-pack/plugins/maps/public/classes/tooltips/join_tooltip_property.ts

```diff
@@ -51,7 +52,8 @@ export class JoinTooltipProperty implements ITooltipProperty {
           this._tooltipProperty.getRawValue()
         );
         if (esTooltipProperty) {
-          esFilters.push(...(await esTooltipProperty.getESFilters()));
+          const filters = await this._tooltipProperty.getESFilters();
```

Comment:
Looks like there is a copy paste error on this line, `this._tooltipProperty` should be `esTooltipProperty`.

Your branch is adding the same filter twice, so one gets removed during merge filters phase.

<img width="300" alt="Screen Shot 2020-10-27 at 7 42 20 AM" src="https://user-images.githubusercontent.com/373691/97309992-580df400-1828-11eb-8057-c725e37e2ff4.png">

On Master, there are 2 different filters.

<img width="300" alt="Screen Shot 2020-10-27 at 7 44 07 AM" src="https://user-images.githubusercontent.com/373691/97310034-6825d380-1828-11eb-801c-f30bff1a711d.png">

Reply:
Oh wow, really glazed over those subtle differences. Thanks!